src/python/grpcio/grpc/_cython/_cygrpc/completion_queue.pyx

```diff
@@ -74,7 +68,41 @@ cdef class CompletionQueue:
     with self.poll_condition:
       self.is_polling = False
       self.poll_condition.notify()
+    return self.interpret_event(event)
+
+  def pluck(self, records.OperationTag tag, records.Timespec deadline=None):
+    # Plucking a 'None' tag is equivalent to passing control to GRPC core until
+    # the deadline.
+    # TODO(atash) there's currently no way of getting a tag to pass to this
```

Comment:
I'm not seeing the tension you describe because I have that dogma that says to never document as part of a method how the method happens to be used by the current code that happens to use it. At this level of abstraction, the method does everything the underlying C `pluck` does. End of story, here, for me, and if there's a drawback or caveat to be documented about the way `next` and `pluck` are used above this, well that can be said there.

(Sorry for all this fuss and bother over a two-line implementation comment.)